### PR TITLE
Update docs to 1.1.0

### PR DESCRIPTION
**Pull request by gabbybilka**

### CODE REVIEW

File: docs/roadmap.md [hunk 1]
```diff
@@ -63,30 +63,31 @@ Blank -  Not available
 
 **WinUI 3 - Windows App SDK 1.0** is our latest release. For a detailed look at the features we're targeting in WinUI 3 in 2022, check out the [Windows App SDK feature roadmap](https://github.com/microsoft/WindowsAppSDK/blob/main/docs/roadmap.md).
```
Comment (bpulliam) [inline docs/roadmap.md:64]: Should also be 1.1 or should not say "is our latest release"

**Review by bpulliam — APPROVED** [review]

**Review by bpulliam — APPROVED** [review]

**Comment by Jay-o-Way** [comment]
Lol you guys are slow, 1.1.0 isn't even the latest anymore

**Pull request merged**